guides/en/scopes.md

```diff
@@ -0,0 +1,32 @@
+# Scopes
+Scopes are regions of a programm or application in which a variable exists. Variables with the same name in different scopes are different variables.
+
+## Variables Scope
+see Variables Scope
+
+## CGI Scope
+see CGI Scope
```

Comment:
Add a link to cgi scope guide.

Reply:
Won't that be done by `autoLink()` function?

Reply:
It will.